database/account.go

```diff
@@ -28,6 +32,19 @@ func (a *Account) TableName() string {
 	return "account"
 }
 
+func (a *Account) String() string {
+	// Convert data to base32 and encode as bech32
+	convData, err := bech32.ConvertBits(a.StakingKey[:], 8, 5, true)
+	if err != nil {
+		return ""
+	}
+	encoded, err := bech32.Encode("stake", convData)
+	if err != nil {
+		return ""
+	}
+	return encoded
+}
```

Comment:
I don't think this is going to do what you think that it is. You probably need to build a `ledger/common.Address` and use its `String()`